app/components/quiz/QuizAnswer.js

```diff
@@ -1,22 +1,49 @@
 import React from 'react';
+import AppStore from 'store/appStore';
+import 'quiz.scss';
+import jquery from 'jquery';
 
 export default React.createClass({
   displayName: 'QuizAnswer',
 
   propTypes: {
-    answer: React.PropTypes.object.isRequired
+    answer: React.PropTypes.object.isRequired,
```

Comment:
this is an awful lot of props for what should be a really simple component - any way to remedy that? you won't need `id` once the jquery code is swapped out with `setState` (see comment below)